pkg/refactor/datastore/datastoresql/sectets.go

```diff
@@ -0,0 +1,91 @@
+package datastoresql
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/direktiv/direktiv/pkg/refactor/core"
+	"github.com/google/uuid"
+	"gorm.io/gorm"
+)
+
+type sqlSecretsStore struct {
+	db *gorm.DB
+}
+
+func (s sqlSecretsStore) CreateFolder(ctx context.Context, namespace uuid.UUID, name string) error {
+	// TODO: yassir: implement me
+	panic("implement me")
+}
+
+func (s sqlSecretsStore) Update(ctx context.Context, secret *core.Secret) error {
+	// TODO yassir: implement me
+	panic("implement me")
+}
+
+func (s sqlSecretsStore) DeleteFolder(ctx context.Context, id uuid.UUID, key string) error {
+	// TODO yassir: implement me
+	panic("implement me")
+}
+
+func (s sqlSecretsStore) Delete(ctx context.Context, namespaceID uuid.UUID, name string) error {
+	res := s.db.WithContext(ctx).Exec(`DELETE FROM secrets WHERE namespace_id = ? and name = ?`,
+		namespaceID, name)
+	if res.Error != nil {
+		return res.Error
+	}
+	if res.RowsAffected != 1 {
+		return fmt.Errorf("unexpedted gorm delete count, got: %d, want: %d", res.RowsAffected, 1)
+	}
+
+	return nil
+}
+
+func (s sqlSecretsStore) Search(ctx context.Context, namespace uuid.UUID, name string) ([]*core.Secret, error) {
+	// TODO yassir: implement me
```

Comment:
I can't remember what the purpose of this API is. Based on the old code, do you know? I can research to refresh my memory if necessary.

Reply:
In the frontend, i didn't find anything related to search. I assumed it is just unused code. Can you check it?

Reply:
I can't recall either. It might have something to do with the directories. Either way, if it's unused I think we can safely ignore it for now.